token-lending/program/tests/helpers/mod.rs

```diff
@@ -465,6 +468,29 @@ pub fn add_reserve(
     }
 }
 
+pub fn add_token_account_for_flash_loan_receiver(
```

Comment:
```suggestion
pub fn add_account_for_program(
```

This could be generalized since the helper implementation isn't specific to flash loans